prime-router/docs/getting-started/fhir-functions.md

```diff
@@ -1,7 +1,85 @@
 # FHIR Functions
+FHIR functions are methods that can be run on a bundle (ex. retrieve an age or telephone area code) via the command line using the fhirpath tool. To use the FHIR 
```

Comment:
`FHIR functions are methods that can be run on a bundle (ex. retrieve an age or telephone area code) via the command line using the fhirpath tool.`

It is my understanding that this statement is true, but that's not what FHIR functions are/their main purpose, so it feels weird to have this as the first sentence? I like how you have internal/external functions broken down below. I would add a third section with a heading like `Prime CLI FHIRPath Tool` and move most/all the content here under there. Then you can have a new introduction up here that talks about FHIR functions generally?

Reply:
An idea for reorg: I would suggest keeping in this section the following:

> FHIR functions are methods that can be run on a bundle. Currently, there are two places where FHIR functions are created. Both are in a file called CustomFhirPathFunctions and
both extend an interface called FhirPathFunctions. There are two places for these functions because some
are internal functions, meaning that they can only be called from within the code rather than from the command line.

Also, I would maybe avoid listing out every function we have, and would link to the code where they are defined. In the docs here I would focus more on what makes a method a fhir function, how do you define one, how do you invoke one, etc. Although I do think one or two examples could be helpful and are appropriate.

Reply:
I thought that was what FHIR functions are and their purpose. Will you please elaborate on what you think should be different?

Reply:
Made some of the changes but I doubt this addresses everything you were wanting. Please lmk about the definition.